services/api/routes/public.js

```diff
@@ -26,6 +26,7 @@ var routes = [
     config: {
       validate: {
         payload: {
+          id: numericSchema.required(),
```

Comment:
Joi converts strings to numbers by default, so I think this can just become `Joi.number().integer().postive()`